src/components/report-components/ReportDescription.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <p v-if="description" class="mg-report-description">
+    <strong>Description: </strong>{{description.substr(0,maxLength)}}
```

Comment:
Descriptions get cut off mid-word, would be cleaner to at the very least do this at word borders.
Once you start making this pretty, it quickly explodes, so a library like https://www.npmjs.com/package/vue-ellipsis is probably even better